src/Raven.Client/Documents/Smuggler/SmugglerResult.cs

```diff
@@ -44,6 +44,8 @@ public SmugglerResult()
             Subscriptions = new Counts();
             ReplicationHubCertificates = new Counts();
             TimeSeries = new CountsWithSkippedCountAndLastEtag();
+            TimeSeriesDeletedRanges = new CountsWithSkippedCountAndLastEtag();
```

Comment:
requires Studio work?

Reply:
Does the studio work need to be part of the same PR or can be handled later on? Will the studio work if we just merge this PR?

Reply:
@avi?

Reply:
yes, studio work can be handled later on